packages/icons/src/at.svg

```diff
@@ -1,5 +1,7 @@
 <svg viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
-	<path
-    d="M16 20C18.2091 20 20 18.2091 20 16C20 13.7909 18.2091 12 16 12C13.7909 12 12 13.7909 12 16C12 18.2091 13.7909 20 16 20ZM16 22C12.6863 22 10 19.3137 10 16C10 12.6863 12.6863 10 16 10C19.3137 10 22 12.6863 22 16C22 19.3137 19.3137 22 16 22Z M16.0313 3C17.049 3.00006 18.5662 3.00255 20.355 3.62721C22.1513 4.25447 24.1567 5.48361 26.2474 7.83564C27.9943 9.80083 28.7064 12.326 28.9123 14.5321C29.1171 16.7262 28.8348 18.7554 28.4398 19.8417C28.2268 20.4274 27.6481 21.076 26.9472 21.5609C26.2079 22.0724 25.196 22.5 24 22.5H23.9839L23.9679 22.4995C23.3504 22.4796 22.4903 22.2785 21.7095 21.6857C20.9046 21.0747 20.2527 20.0987 20.0136 18.6644L20 18.5828V11.5C20 10.9477 20.4477 10.5 21 10.5C21.5523 10.5 22 10.9477 22 11.5V18.4131C22.169 19.3288 22.5576 19.8186 22.9188 20.0927C23.3069 20.3874 23.7382 20.488 24.0171 20.5C24.7206 20.4959 25.3369 20.243 25.8093 19.9162C26.0467 19.7519 26.2362 19.5766 26.3709 19.4226C26.5134 19.2595 26.5584 19.1622 26.5602 19.1584C26.8318 18.4114 27.1051 16.6904 26.921 14.7179C26.738 12.7574 26.1169 10.6992 24.7526 9.16436C22.864 7.03975 21.1377 6.01895 19.6956 5.5154C18.2462 5.00926 17.0186 5.00006 16.0313 5M16.0313 3L16 3L16.0313 3ZM16.0313 5C13.5386 5.00002 8.09122 6.32574 5.91914 11.3939L5.91373 11.4066L5.90796 11.4191C5.46939 12.3693 5 14.1011 5 16C5 17.8971 5.46812 19.9286 6.36824 21.5039C6.97566 22.5668 8.8683 24.9984 11.3939 26.0809C14.0295 27.2104 16.9449 27.445 19.1286 26.5715C19.6414 26.3664 20.2234 26.6158 20.4285 27.1286C20.6336 27.6414 20.3842 28.2234 19.8714 28.4285C17.0551 29.555 13.5705 29.1896 10.6061 27.9191C7.53171 26.6016 5.35768 23.7665 4.63176 22.4961C3.53188 20.5714 3 18.1862 3 16C3 13.8231 3.52694 11.8114 4.0862 10.5936C6.70796 4.49451 13.1248 3.01068 16 3"
+<path
+    fill-rule="evenodd"
```

Comment:
why all these `fill-rule` and `clip-rule`?

Reply:
If I'm not wrong, this was introduced by [Fill Rule](https://www.figma.com/community/plugin/771155994770327940/fill-rule-editor) in figma to fix some svg problems related to that... 🤔